recipes/coverage-fixpaths/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "coverage-fixpaths" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18880405439c1a6329e5448b104c2b03de248fa4f54a7ec2f123fd226a2a573b
+
+build:
+  number: 0
+  skip: true  # [win or py<36]
```

Comment:
The meta data in the `setup.py` says this package is os independent. Is there are reason to skip windows? If not, I think it would be preferable to remove the `skip` line, make this `noarch: python` and then change `python` to `python >=3.6` for `host` and `run`.

Reply:
I had that previously, but then [the build failed on Windows(https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=150230&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=78a0099b-3f13-5ae9-c1f0-24f71d1eac20).
I've applied the suggestion, hoping that the new entrypoint fixes that.